scripts/rust-build.sh

```diff
@@ -65,15 +65,15 @@ fi
 TARGET=wasm32-unknown-unknown
 WASM_BINARY=$SELF_DIR/target/$TARGET/release/${PNAME}.wasm
 
-CJS_DIR=$SELF_DIR/dist/cjs/wasm/
-ESM_DIR=$SELF_DIR/dist/esm/wasm/
-CJS_WASM=${CJS_DIR}/${PNAME}_bg.wasm
+#CJS_DIR=$SELF_DIR/dist/cjs/wasm/
+ESM_DIR=$SELF_DIR/src.ts/wasm/
+#CJS_WASM=${CJS_DIR}/${PNAME}_bg.wasm
```

Comment:
TODO: this around here would for sure need still some clean-up if we go this way.